<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExistDialog</class>
 <widget class="QDialog" name="ExistDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fingerprint exists</string>
  </property>
  <property name="windowIcon">
   <iconset resource="widget-res.qrc">
    <normaloff>:/new/prefix1/res/fingerprint-gui.png</normaloff>:/new/prefix1/res/fingerprint-gui.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="fingerLabel">
         <property name="text">
          <string>A fingerprint for Left Little exists already.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>You can verify this fingerprint or remove it and acquire new.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="verifyButton">
       <property name="text">
        <string>&amp;Verify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acquireButton">
       <property name="text">
        <string>&amp;Acquire new</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="widget-res.qrc"/>
 </resources>
 <connections/>
</ui>
